public/downloads/source-code/index.html

```diff
@@ -170,12 +170,12 @@
 				<h1 class="title">Source Code</h1>
 				
 								
-				<!--Searching /home/www/shutter/wordpress/wp-content/uploads/random: found 78 images in 0.00028600000000001 seconds-->
-<!---Displayed in 0.008367 seconds.-->
-<p>Latest source packages are <a href="/wp-content/uploads/releases/tars">available here</a>. If you build a package for your distribution, please <a href="../contact">let us know</a> &#8211; we&#8217;ll link to it.</p>
-<h5>Latest code snapshot:</h5>
-<p>If you are interested in the latest snapshot you can checkout the dev-branch from our bazaar repository:</p>
-<p><a class="piwik_download" href="https://code.launchpad.net/~mario-kemper/shutter/devel">https://code.launchpad.net/~mario-kemper/shutter/devel</a></p>
+				<!--Searching /home/www/shutter/wordpress/wp-content/uploads/random: found 78 images in 0.00028600000000001 seconds-->
+<!---Displayed in 0.008367 seconds.-->
+<p>Latest source packages are <a href="https://launchpad.net/shutter/+download">available here</a>. If you build a package for your distribution, please <a href="https://shutter-project.org/contact/">let us know</a> &#8211; we&#8217;ll link to it.</p>

```

Comment:
Can we change the link to `/contact` to remove the domain specific link?

Reply:
I fixed it in the forked branch, now not sure how to update the PR (and if it is necessary at all). Sorry, don't have any much experience with PRs so far. :)